### PR TITLE
[Remove Stream/Optional usage] LPS-171138 export-import

### PR DESCRIPTION
**Pull request by JanisZhang**
Remove usages of Stream on Method Return Value for module: `export-import`.

https://issues.liferay.com/browse/LPS-171138

### CODE REVIEW

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lifecycle/EventRemotePropagatorExportImportLifecycleListener.java [hunk 1]
```diff
@@ -92,33 +90,41 @@ private boolean _eventNeedsToBePropagated(
 			return false;
 		}
 
-		Optional<ExportImportConfiguration> exportImportConfigurationOptional =
+		ExportImportConfiguration exportImportConfiguration =
```
Comment (dantewang) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lifecycle/EventRemotePropagatorExportImportLifecycleListener.java:93]: @JanisZhang As we talked, please update this part.

**Comment by JanisZhang** [comment]
force-pushed. Thanks!

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:export-import

**Comment by Kvale1733** [comment]
ci:test:export-import

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lifecycle/EventRemotePropagatorExportImportLifecycleListener.java [hunk 2]
```diff
@@ -138,48 +144,38 @@ private boolean _eventNeedsToBePropagated(
...
-				exportImportConfiguration.getSettingsMap()
-		).map(
-			settingsMap -> MapUtil.getLong(settingsMap, "userId")
-		).map(
-			userId -> _userLocalService.fetchUser(userId)
-		).flatMap(
-			user -> _getHttpPrincipal(
-				user, _getRemoteURL(exportImportLifecycleEvent))
-		);
+		return _getHttpPrincipal(
```
Comment (dantewang) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lifecycle/EventRemotePropagatorExportImportLifecycleListener.java:171]: @JanisZhang Please also fix this one.

Basically, each `.map` (or maybe `.flatMap` also) should be converted into if-null-return-null. For example, what if the `settingsMap` doesn't contain `userId`? The local service will query the user with `0`, which may return null.

If user is null, the original `_getHttpPrincipal` method will return `Optional.empty()`, while the changed version will throw NPE.

cc @W2jx @slnn @yuhai @LisztLiszt

**Review by dantewang — CHANGES_REQUESTED** [review]
Please see my comment and resend.

**Pull request closed without merging**